packages/block-editor/README.md

```diff
@@ -76,6 +76,10 @@ registerCoreBlocks();
 
 <!-- START TOKEN(Autogenerated API docs) -->
 
+<a name="AlignmentControl" href="#AlignmentControl">#</a> **AlignmentControl**
```

Comment:
What's the story moving forward with all pairs like `AlignmentControl` and `AlignmentToolbar`. It can quickly become confusing what should be used when developing a block.

Reply:
I'd love to drop the "Toolbar" ones but you know BC :)

Reply:
I can deprecate these once we update all the blocks.

Reply:
That's the answer I wanted to hear. It makes a lot of sense to approach it this way 👍🏻